src/content/docs/style-guide/writing-strategies/contribute-to-site.mdx

```diff
@@ -0,0 +1,376 @@
+
+<Callout variant="tip">
+If you're a New Relic employee, we recommend joining the New Relic org on GitHub. To join the org, ping the @hero at #help-opensource on Slack. 
+</Callout>
+
+## Contribute to the docs site [#contribute-work]
+
+You don't need a complete understanding of git or GitHub to get set up, but you might encounter some unfamiliar terms:   
+
+* A **commit** acts like a save. 
+* A **branch** houses your commit(s).
+* A **pull request** packages your branch and sends it to us for review. 
+
+<Tabs>
+	<TabsBar>
+        <TabsBarItem id="edit-browser">
+            Edit via the browser
+        </TabsBarItem>
+        <TabsBarItem id="copy-repo">
+            Copy the site to your GitHub account
+        </TabsBarItem>
+        <TabsBarItem id="request">
+            Make a request to the team 
+        </TabsBarItem>
+  </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="edit-browser">
+
+For most cases, we recommend editing the docs site via your browser. On every page, you can click **Edit this doc** to edit directly from your browser. 
+
+<Steps>
+    <Step>
+    ### Click **Edit this doc** 
+
+On the doc you'd like to edit, click **Edit this doc** from the right nav. This opens a text view of the doc in question, letting you view the copy alongside its code. 
+    
+  <img
+    title="Make small changes to a doc"
+    alt="Click the edit button to make small changes"
+    src={styleguideEditDoc}
+  />
+
+    </Step>
+    <Step>
+        ### Make your edits 
+
+Once you've opened the text-version of a doc, find the **pencil** icon in the top nav of the text element. This lets you interract with the doc in edit-mode rather than the default view-only mode. 
+
+    <img
+    title="Click the pencil button"
+    alt="Click the pencil button to go into edit mode"
+    src={styleguidePencilButton}
+    /> 
+
+Make your edits, and don't worry too much about formatting or grammar. We've got that covered.
+
+    </Step>
+    <Step>
+        ### Apply your changes
+
+  <img
+      title="Commit your changes"
+      alt="Click the pencil button to go into edit mode"
+      src={styleguideCommitChanges}
+  />
+
+Once you're ready to save your progress, click the **Commit changes...** button. This brings you to a page that looks like this:
+
+  <img
+    title="Write a commit message"
+    alt="Click the pencil button to go into edit mode"
+    src={styleguideCommitMessage}
+  />
+
+Let's break down what this form is asking:
+
+* **Commit message**: Required. A commit message is a brief explanation of what you changed. For example, one might read like: `Updated requirements and added content to install procedures`
```

Comment:
Can we give an example that's more like what our team would want to see? Like, is this a "good" commit message for our team? Would we prefer something else?